### ISSUE TITLE
Error during run on windows 11

### ISSUE THREAD

**Issue by Landelor** [rcdailey/trash-updater#15]
Hi,

I'm currently getting this error when trying to run the exe in any command line:
```
Error:
  An assembly specified in the application dependencies manifest (trash.deps.json) was not found:
    package: 'LibGit2Sharp.NativeBinaries', version: '2.0.312'
    path: 'runtimes/win-x64/native/git2-6777db8.pdb'
```

Running latest version 1.6.2

Regards,
Landelor

**Comment by Landelor** [comment]
Retested previous version and that works fine.

**Comment by rcdailey** [comment]
What OS (and OS version) are you running on?

**Comment by Landelor** [comment]
Was Windows 11, and I'm assuming that is the issue. I'm in the process of reverting and will retest once done.

**Comment by Mini-Myself** [comment]
Same thin here on Windows 10 21H1 19043.1110

**Comment by rcdailey** [comment]
This is now fixed in [v1.6.3](https://github.com/rcdailey/trash-updater/releases/tag/v1.6.3)

Note that you will temporarily see a new PDB file bundled with `trash.exe`. This is normal and that file is required. The release notes go into more details.
